renew_certificate.sh

```diff
@@ -51,9 +51,11 @@ echo "Started python HTTP server with pid $pid"
 
 export SSL_CERT_FILE=cacert.pem
 "$PYTHON" acme-tiny/acme_tiny.py --account-key letsencrypt/account.key --csr letsencrypt/domain.csr --acme-dir tmp-webroot/.well-known/acme-challenge > letsencrypt/signed.crt.tmp
-mv letsencrypt/signed.crt.tmp letsencrypt/signed.crt
+if [ -f letsencrypt/signed.crt.tmp ] && [ -s letsencrypt/signed.crt.tmp ]; then
+    mv letsencrypt/signed.crt.tmp letsencrypt/signed.crt
+fi
```

Comment:
please you undo this change

Reply:
I can remove that but why?

I had some problems with letsencrypt Not issuing certificates. In that case
the resulting signed.crt.tmp is not there or empty. The only thing that if
does, is checking if the file exists and is not empty. So it is just makes
sure the renewed certificate is valid before replacing the probably still
working one.

I am very interested to hear why this change should be removed.

Reply:
we can talk about it in a separate issue. it's a change completely unrelated to the update of the intermediate cert.